### DISCUSSION TITLE
create getter for relation in lucid model

### DISCUSSION THREAD

**Discussion by farshadfahimi** [adonisjs/core#1510] [Help]
I have a model like below

```javascript
class User extends Model
{
    static get computed() {
        'carsCount'
   }

   getCarsCount() {
       return this.cars().getCount()
   }

    cars() {
        return this.hasMany('App/Models/Car')
    }
}
```

If I create the model like above every time I'm fetching the user `computed` property call and start query on the database for fetch the count of user cars.

I know, I can use `withCount` in controller to add `carsCount` to the model in controller
but I'm researching to find a light way to do this in the `model`

do you have any suggestion for me to use `getter` for this ?

**Comment by arthurcvm** [comment]
Hi! You solve this?
